product_docs/docs/pgd/5/terminology.mdx

```diff
@@ -88,11 +125,12 @@ A multi-step process for achieving consistency across multiple database nodes.
 
 A traditional computing approach of increasing a resource (CPU, memory, storage, network) to support a given workload until the physical limits of that architecture are reached, e.g., Oracle Exadata.
 
-#### Write scalability
+#### Witness nodes
+
+To resolve clusters or groups of nodes coming to a consensus, there needs to be an odd number of data nodes. Where resources are limited, a witness node can be used to create an odd number of data nodes, participating in cluster decisions but not replicating the data. Not holding the data means it is not able to operate as a standby server.
```

Comment:
@djw-m thinking rearranging might help? i.e. 

"Witness nodes primarily serve to help with consensus. Where resources are limited, a witness node can be used to create an odd number of data nodes, participating in cluster decisions but not replicating the data. Not holding the data means it is not able to operate as a standby server."

Reply:
```suggestion
Witness nodes primarily serve to help the cluster establish a consensus. An odd number of data nodes are needed to establish a consensus and, where resources are limited, a witness node can be used to participate in cluster decisions but not replicate the data. Not holding the data means it cannot operate as a standby server or provide majorities in synchronous commits.
```